pkg/controller/core/clusterqueue_controller.go

```diff
@@ -80,6 +82,7 @@ func (r *ClusterQueueReconciler) Reconcile(ctx context.Context, req ctrl.Request
 	ctx = ctrl.LoggerInto(ctx, log)
 	log.V(2).Info("Reconciling ClusterQueue")
 
+	const cqTerminatingStatusMsg = "Can't admit new workloads; clusterQueue is terminating"
```

Comment:
put the constant as a package variable on top of the file

Reply:
Ok.

Reply:
I removed this constant since I removed this logic for the following reason.

> We don't need the logic here.
>
> If we are able to remove the finalizer below, then this object will be deleted. Then adding the condition is just a waste.

https://github.com/kubernetes-sigs/kueue/pull/415#discussion_r1043701123